### PR TITLE
fix : inconsistent styling in Events section

### LINKED ISSUE #8004: [BUG] UI Improvements in Form
### Description

There are some style mismatch which can be improved to maintain consistency.

### Screenshots

Such as :- 

image

image

### Additional information

I would like to work on the same.

### PR DESCRIPTION
**Pull request by yoyoekta**
## Fixes Issue

closes #8004 

## Changes proposed

Removed inconsistency from Events section.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Changes :-
image

image

Reference :- 
image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8022"></a>

### CODE REVIEW

File: components/user/UserEvents.js [hunk 1]
```diff
@@ -64,28 +64,28 @@ export default function UserEvents({ manage = false, events }) {
...
-      {!manage && (
-        <DropdownMenu
-          eventType={eventType}
-          handleEventTypeChange={handleEventTypeChange}
-          options={filteredEventOptions}
-          label="Select Event Type:"
-          className="inline text-center text-sm font-medium leading-6 text-primary-high sm:pt-1.5"
-        />
+    <>
+      {!eventsToShow?.length && (
```
Comment (eddiejaoude) [inline components/user/UserEvents.js:68]: I find this much easy to read than using the else `:` condition 👍

**Comment by yoyoekta** [comment]
Asking for re-review was accidental. Kindly, ignore it.

**Comment by eddiejaoude** [comment]
> Asking for re-review was accidental. Kindly, ignore it.

lol ok

**Review by eddiejaoude — COMMENTED** [review]
Thank you 👍

**Pull request merged**